app/controllers/application_controller/timelines.rb

```diff
@@ -127,41 +127,12 @@ def tl_chooser
     end
   end
 
-  def getTLdata
-    if session[:tl_xml_blob_id] != nil
-      blob = BinaryBlob.find(session[:tl_xml_blob_id])
-      render :xml=>blob.binary
+  def timeline_data
```

Comment:
If this `timeline_data` method is in ApplicationController, why do we need the duplicate methods in subclassed controllers (DashboardController and MiqCapacityController)?